releasenotes/notes/tag_by_endpoint-support-in-prometheusScrape-47d13a051221057d.yaml

```
@@ -0,0 +1,3 @@
+enhancements:
+  - |
+    prometheusScrape: Adds support for `tag_by_endpoint`
```

the description can be more precise

```suggestion
    prometheus scrape: Adds support for `tag_by_endpoint` in the `prometheus_scrape.checks[].configurations[]` items.
```

prometheusScrape is the naming in the helm-chart but, in the datadog-agent config the proper config keyword is `prometheus_scrape` see: https://github.com/DataDog/datadog-agent/blob/6683a54acf0f85de6a536898ec1a5afcab0ad0b8/pkg/config/config.go#L651

---

Hey, the  `collect_counters_with_distributions` was added at line `102` (in the Openmetrics V2 specifics) which needed to align the whole block since it's 1 character longer than the previous longer parameter.
Updated the release note based on your suggestion

---

Hi @tbavelier,
Sorry I didn't saw the `collect_counters_with_distributions` during the first review due to the block indentation update.